src/NServiceBus.Transport.RabbitMQ.CommandLine/Commands/Endpoint/EndpointMigrateCommand.cs

```diff
@@ -118,6 +118,11 @@ void ConvertToQuorum(IModel channel, CancellationToken cancellationToken)
 
             console.WriteLine($"{numMessagesMovedToHolding} messages moved to the holding queue");
 
+            if (channel.MessageCount(queueName) > 0)
```

Comment:
In theory we could spin for some period of time but I think this is good enough. We should spell out clearly that direct sends (since we only support the conventional routing topology it would have to be direct sends from native integrations) are not supported in the docs